### PR TITLE
[ADD][BR1030][22330]-Added Business requirement etherpad module

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
Issue #110 

- Added a field tab "Notes" (before logs) with a field notes link to Etherpad.

### CODE REVIEW

File: business_requirement_etherpad/README.rst [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+=============================
+Business Requirement Etherpad
+=============================
+
+
+Configuration
```
Comment (elicoidal) [inline business_requirement_etherpad/README.rst]: "This module extends the functionality of the business requirements and introduces a field link to Etherpad in order for the team to keep internal notes about the BR"

Remove configuration and Usage content.

File: business_requirement_etherpad/README.rst [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eric Caudal <eric.caudal@elico-corp.com>
```
Comment (elicoidal) [inline business_requirement_etherpad/README.rst:39]: Keep only myself

File: business_requirement_etherpad/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Etherpad',
+    'category': 'Business Requirements Management',
+    'summary': 'Manage the Business Requirements Notes',
+    'version': '8.0.5.1.3',
```
Comment (elicoidal) [inline business_requirement_etherpad/__openerp__.py]: 8.0.1.0.0

File: business_requirement_etherpad/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Etherpad',
+    'category': 'Business Requirements Management',
+    'summary': 'Manage the Business Requirements Notes',
```
Comment (elicoidal) [inline business_requirement_etherpad/__openerp__.py]: Manage the Business Requirements Notes via Etherpad

**Review by elicoidal — CHANGES_REQUESTED** [review]
improve the README and manifest.
Other than that, LGTM

File: business_requirement_etherpad/README.rst [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+Followers from the project will automatically be added to the business requirements.
+
+
+Usage
+=====
+Simple BR
+---------
+
+1. Define the header information
+
```
Comment (serpentcs-dev1) [inline business_requirement_etherpad/README.rst]: @elicoidal What do you think about the Usage? What should be included?

Right I think it is not correct.

Comment (elicoidal) [reply business_requirement_etherpad/README.rst]: I think in usage a simple screenshot with the field and some lorem ipsum text is good enough.
@darshan-serpent

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@victormartinelicocorp

**Comment by martintorresvm** [comment]
@elicoidal I had to check the Issue https://github.com/OCA/business-requirement/issues/110 and makes sense then.

**Review by martintorresvm — APPROVED** [review]

**Comment by elicoidal** [comment]
@dreispt

**Pull request merged**